_data/hotels.yml

```diff
@@ -1,9 +1,11 @@
 websites:
   - name: Airbnb
     url: https://www.airbnb.com
-    twitter: Airbnb
-    facebook: airbnb
     img: airbnb.png
+    tfa:
+      - email
+      - sms
+    doc: https://www.airbnb.com/help/article/501
```

Comment:
The doc link doesn't describe how to enable 2FA.

Reply:
The doc page (kind of) suggests that it is enabled by default. There is a [reddit comment](https://www.reddit.com/r/AirBnB/comments/865zj2/how_to_enable_2fa/) by a "Verified Airbnb" user that would confirm this.

In the original PR #4238, @kmpoppe noted that he didn't get any verfication message. Maybe this is dependent on the user's region?